tests/misc/test_basic_without_ssh.py

```diff
@@ -72,6 +73,18 @@ def test_checkpoint(self, imported_vm):
         finally:
             snapshot.destroy(verify=True)
 
+    def test_fileserver_redirect_https(self, host):
+        path = "/path/to/dir/file.txt"
+        process = subprocess.Popen(
+            ["curl", "-i", "http://" + host.hostname_or_ip + path],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE
+        )
+        stdout, stderr = process.communicate()
+        lines = stdout.decode().split('\r\n')
```

Comment:
I think this is equivalent in this context and more elegant: https://docs.python.org/3/library/stdtypes.html#str.splitlines

```suggestion
        lines = stdout.decode().splitlines()
```